procedures/katello/branch.md.erb

```diff
@@ -3,25 +3,41 @@
 ## Release Owner
 
 - [ ] Announce upcoming branching to discourse development category a month before
-- [ ] Start extracting i18n strings and pushing the changes into master git branches so Transifex can pick it up
 - [ ] Update Katello Transifex translations:
   - [ ] Create a Transifex account and join the Foreman team
   - [ ] Spin up a Foreman and Katello installation
-  - [ ] Install Transifex client `pip3 install transifex-client --user`
-  - [ ] [Configure the Transifex client](https://projects.theforeman.org/projects/foreman/wiki/Translating#How-to-pull-translations)
+  - [ ] Install Transifex client: `pip3 install transifex-client --user` (make sure to pass `--user` flag!)
+  - [ ] Configure `~/.transifexrc` [as outlined here](https://projects.theforeman.org/projects/foreman/wiki/Translating#How-to-pull-translations)
   - [ ] [Install grunt](https://github.com/Katello/katello/blob/master/engines/bastion_katello/README.md)
   - [ ] `grunt i18n:extract` in the katello/engines/bastion_katello directory
   - [ ] `make -C locale tx-update` in the katello directory
   - [ ] `grunt i18n:compile` in the katello/engines/bastion_katello directory
   - [ ] `bundle exec rake plugin:gettext[katello]` in the foreman directory
   - [ ] Commit the resulting files in the katello directory with a message like "i18n - pulling from tx"
+  - [ ] Open a PR to Katello (no Redmine issue needed)
+- [ ] Start attending upstream release sync meetings and giving updates
+
+# Three Weeks Prior to Branch Date
+
+## Release Owner
+
+- [ ] Ensure that issues requiring installer changes are merged
 
 # Two Weeks Prior to Branch Date
 
 ## Release Owner
 
-- [ ] Add tool_belt config for new release [tool_belt configs](https://github.com/theforeman/tool_belt/tree/master/configs/katello)
-  - [ ] Check the nightly config file if any repos/tags need to be updated in the newly created config.
+- [ ] Add tool_belt config for new release:
+  - [ ] Create a new yaml file using the previous Katello version as a template: [tool_belt configs](https://github.com/theforeman/tool_belt/tree/master/configs/katello)
+  - [ ] Manually update the following sections:
+    - [ ] releases: update to current release.  Move the previous 'current' release to prior_releases below.
+    - [ ] prior_releases: Remove the oldest prior_release (check with that release owner first to see if there's a reason it should stay)
+    - [ ] mash_scripts: update Katello version number in all values
+    - [ ] repos: Update branch names to current versions, including any new releases that need to happen
+    - [ ] ignores: Delete all items from this list and start fresh (this will be used for cherry-picks later)
+    - [ ] gpg-key: Get the new Foreman GPG key when available ([example here](https://github.com/theforeman/theforeman.org/pull/1821)) and put the last 8 digits here
```

Comment:
https://github.com/theforeman/theforeman-rel-eng may be a good source? For example, the [Foreman 2.5 settings](https://github.com/theforeman/theforeman-rel-eng/blob/master/releases/foreman/2.5/settings) has the full GPG key. We then transform this into the correct format [here](https://github.com/theforeman/theforeman-rel-eng/blob/75d8fa9f58784e2ca7f334463b23d4077d1d8a96/settings#L67-L68).